### PR TITLE
Document OAuth 2/UAA support in management plugin

### PR DESCRIPTION
**Pull request by acogoluegnes**
[#169725428]

### CODE REVIEW

**Comment by dcorbacho** [comment]
@acogoluegnes Should we show/explain the redirect and logout? Also how tokens are sent on the HTTP API?

**Comment by acogoluegnes** [comment]
> @acogoluegnes Should we show/explain the redirect flow and logout?

Not sure it's necessary, the page is already quite long.

> Also how tokens are sent on the HTTP API?

Can you be more specific? I don't remember there's something special regarding the HTTP API.

**Comment by dcorbacho** [comment]
Almost all requests use in the authorization header `Bearer <token>' as expected, but the `definitions` import/export endpoint needs to use `token=<token>` in the query string.

**Comment by acogoluegnes** [comment]
> Almost all requests use in the authorization header `Bearer <token>` as expected, but the `definitions` import/export endpoint needs to use `token=<token>` in the query string.

This is when HTTP basic authentication is disabled, right? This is worth mentioning indeed, I'll add it.

File: site/management.md [hunk 1]
```diff
@@ -275,6 +275,82 @@ rabbitmqctl set_user_tags full_access administrator
 </pre>
 
 
+## <a id="http-api" class="anchor" href="#http-api">Authenticating with OAuth 2</a>
```
Comment (michaelklishin) [inline site/management.md]: The anchor here is a copy-paste artifacts and it break on page navigation.

Comment (acogoluegnes) [reply site/management.md]: Good catch, fixed.

**Pull request merged**